build-phar.sh

```diff
@@ -8,10 +8,4 @@ fi
 
 composer install --no-dev --optimize-autoloader
```

Comment:
you shouldn't need that line: Box dump the optimized autoloader and already skips the dev deps